src/index.js

```diff
@@ -95,18 +95,17 @@ export default class Snuffles {
 
         return res
       })
-      .then(res => {
+      .then(async res => {
         const { status } = res
         const resultBase = { status, headers: res.headers, body: {} }
 
-        const contentLength = res.headers['map']['content-length']
-        const contentType = res.headers['map']['content-type']
-
-        if (contentLength <= 1 || contentType !== 'application/json') {
-          return resultBase
+        try {
+          const parsedRes = await res.json()
+          resultBase.body = parsedRes
+        } catch (error) {
+          resultBase.body = {}
```

Comment:
would it make sense to log something in case of an error?

Reply:
Yes, indeed!

Reply:
I'm not too deep into the whole logger system, would we just want it to be logged under `response` (i.e. `this.log('response', error)`) or would it we a dedicated key for response parsing (something like `this.log('response parser', error)`)?